src/VisualStudio/Core/Def/Implementation/FindUsages/ItemOriginColumnDefinition.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio.Composition;
+using Microsoft.VisualStudio.Shell.TableControl;
+using Microsoft.VisualStudio.Shell.TableManager;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.FindUsages
+{
+    /// <summary>
+    /// Custom column to display the item's origin for the Find All References window.
+    /// </summary>
+    [Export(typeof(ITableColumnDefinition))]
+    [Name(ColumnName)]
+    internal class ItemOriginColumnDefinition : TableColumnDefinitionBase
+    {
+        public const string ColumnName = StandardTableKeyNames.ItemOrigin;
```

Comment:
So the constant is defined in StandardTableKeyNames but we have to still do this ITableColumnDefinition?

Reply:
I am working on adding the column definitions to shell.

Reply:
https://github.com/CyrusNajmabadi/roslyn/pull/11 will remove this file